src/views/pages/GateKeeper/index.vue

```diff
@@ -1,38 +1,40 @@
-<template>
-  <div class="content">
-    <md-progress-spinner class="md-accent" :md-stroke="3" md-mode="indeterminate">
-    </md-progress-spinner>
-  </div>
-</template>
-
-
-<script lang="ts">
-import Vue from 'vue';
-
-export default Vue.extend({
-  name: 'GateKeeper',
-  data () {
-    return {
-    };
-  },
-  created(){
-    // TODO update this when we use cookies
-    if (localStorage.getItem("token")) {
-      this.$router.push({ path: 'home'});
-    } else {
-      this.$router.push({ path: 'login'});
-    }
-    
-  },
-});
-</script>
-
-<style lang="scss" scoped>
-.content {
-  width: 100%;
-  height: 100%;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-}
-</style>
+<template>
+  <div class="content">
+    <md-progress-spinner class="md-accent" :md-stroke="3" md-mode="indeterminate">
+    </md-progress-spinner>
+  </div>
+</template>
+
+
+<script lang="ts">
+import Vue from 'vue';
+
+import TokenService from '@/services/Token';
+
+export default Vue.extend({
+  name: 'GateKeeper',
+  data () {
+    return {
+    };
+  },
+  created(){
+    // TODO update this when we use cookies
```

Comment:
remove comment? also since youre here, can you change how we route and use the name instead? 

`this.$router.push({ name: 'home' });`